Loggie/Classes/Core/LogList/LogListTableView.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  LogListTableView.swift
+//  Loggie
+//
+//  Created by Damjan Dimovski on 14.10.22.
+//
+
+import SwiftUI
+
+@available(iOS 13.0, *)
+public struct LogListTableView: UIViewControllerRepresentable {
+
+    let filter: ((Log) -> Bool)?
+
+    public init(filter: ((Log) -> Bool)? = nil) {
+        self.filter = filter
+    }
+
+    public typealias UIViewControllerType = UINavigationController
+
+    public func makeUIViewController(context: Context) -> UINavigationController {
+        let nav = LoggieManager.shared.showLogs(filter: filter)
+        return nav
```

Comment:
I would suggest trying to avoid short naming like `nav` or `vc`.
Here, there is no need to create a variable at all, you can go easily with:
```swift
LoggieManager.shared.showLogs(filter: filter)
```